plugins/inventory/aws_ec2.py

```diff
@@ -64,6 +64,14 @@
         type: str
         default: '_'
         required: False
+  allow_duplicated_hosts:
+    description:
+      - By default, only the first name that back the I(hostnames) list is returned.
```

Comment:
```suggestion
      - By default, only the first name that appears in the I(hostnames) list is returned.
```
Do I understand your meaning here?

Reply:
I should have said "matches".

By default, the first name that matches  one of the entry of the I(hostnames) list is returned.